index.html

```diff
@@ -46,30 +48,256 @@ <h2 id="jobTitle">I am a Software Developer</h2>
+                <path
+                    d="M102.318 201.888L102.549 233.604L71.6522 258.502L71.8294 225.451L94.3899 206.975L102.318 201.888Z"
+                    fill="#262626" />
+                <path d="M137.853 337.713L172.542 328.098L178.634 350.077L143.946 359.692L137.853 337.713Z" fill="white"
+                    stroke="#FF6B00" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path
+                    d="M169.818 341.249L166.012 344.734L165.768 343.853L168.69 341.295L168.674 341.24L164.853 340.551L164.609 339.67L169.665 340.699L169.818 341.249ZM159.817 338.746L160.56 338.54L165.169 346.346L164.426 346.552L159.817 338.746ZM156.434 347.626L152.497 345.398L154.725 341.461L155.401 341.844L153.823 344.623L159.472 343.057L159.682 343.814L154.033 345.38L156.818 346.957L156.434 347.626ZM146.304 347.767L146.151 347.216L149.957 343.731L150.201 344.612L147.28 347.17L147.295 347.225L151.117 347.915L151.361 348.795L146.304 347.767Z"
+                    fill="black" />
+                <path d="M94.3418 199.989L94.4694 231.733L63.5724 256.632L63.7594 226.023L94.3418 199.989Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path d="M138.699 173.915L135.892 144.135L166.817 119.34L169.281 147.881L138.699 173.915Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path
+                    d="M152.591 151.842C150.652 152.379 148.645 151.244 148.108 149.307C147.571 147.369 148.707 145.363 150.646 144.825C152.585 144.288 154.592 145.423 155.129 147.361C155.666 149.298 154.53 151.305 152.591 151.842Z"
+                    fill="white" stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path d="M79.7207 237.489L79.7188 248.712L68.8173 257.512L68.9336 246.702L79.7207 237.489Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
+                <path d="M119.26 254.197L119.351 272.173L102.444 282.971L102.525 265.614L119.26 254.197Z" fill="white"
+                    stroke="#262626" stroke-width="0.496227" stroke-miterlimit="10" />
             </g>
             <defs>
-            <clipPath id="clip0_48_297">
-            <rect width="406.889" height="312.031" fill="white" transform="matrix(-0.963665 0.267112 0.267112 0.963665 392.104 0)"/>
-            </clipPath>
+                <clipPath id="clip0_48_297">
+                    <rect width="406.889" height="312.031" fill="white"
+                        transform="matrix(-0.963665 0.267112 0.267112 0.963665 392.104 0)" />
+                </clipPath>
             </defs>
-            </svg>
-            
-        </main>
+        </svg>
+    </main>
+    <!-- Project/works Section -->
+    <section class="work-section-grid">
+        <div class="recent-work grid-content">
+            <div id="topic">
+                <p>My Recent Works</p>
```

Comment:
- [ ] Please make sure to use an `hx` here, as this is the start of a section, so should include a title and not a paragraph.